bin/qpc-tools

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+#
+"""Script for running qpc command line tool"""
```

Comment:
We should probably call this qpc-tools instead of qpc.

Reply:
yep